docs/user/commandline_tools.rst

```diff
@@ -424,6 +426,114 @@ Explain how a given GeoMesa query will be executed::
       -c test_catalog -f test_feature \
       -q "INTERSECTS(geom, POLYGON ((41 28, 42 28, 42 29, 41 29, 41 28)))"
 
+stats-analyze
+~~~~~~~~~~~~~
+
+Analyze statistics for your data set. This may improve query planning.
+
+Example usage::
+
+    $ geomesa stats-analyze -u username -p password -c geomesa.data -f twitter
+      Running stat analysis for feature type twitter...
+      Stats analyzed:
+        Total features: 8852601
+        Bounds for geom: [ -171.75, -45.5903996, 157.7302, 89.99997102 ]
+        Bounds for dtg: [ '2016-02-01T00:09:12.000Z' to '2016-03-01T00:21:02.000Z' ]
+        Bounds for user_id: [ '100000215' to '99999502' ]
```

Comment:
When I run the stats-analyze command I see the cardinality appended at the end of each line. I assume that is the count of unique attribute values. Maybe document it here?